tests/lib/mongo/repositories/users.test.ts

```diff
@@ -0,0 +1,78 @@
+import { config } from 'dotenv';
+import { MongoDBBucketsRepository } from '../../../../lib/core/buckets/MongoDBBucketsRepository';
+import { bucketsFixtures } from '../fixtures/buckets.fixtures';
+import { unloadLoadFixtures } from '../fixtures/init-fixtures';
+import { setupAndValidateStorageForFixtures } from './utils';
+
+config();
+
+const { storage, uri, BRIDGE_TEST_DB_NAME } =
+  setupAndValidateStorageForFixtures();
+
+let repository: MongoDBBucketsRepository = new MongoDBBucketsRepository(
+  storage.models.User
+);
+
+const [user1, user2] = bucketsFixtures;
```

Comment:
this seems to be a bucket, is preferrable to call it `bucketOne`, `bucketTwo`

Reply:
fixed